provider/source/data/debug/time_zone/primary_zones@1.json

```diff
@@ -0,0 +1,228 @@
+{
+  "adalv": "AD",
+  "aedxb": "AE",
+  "afkbl": "AF",
+  "aganu": "AG",
+  "aiaxa": "AI",
+  "altia": "AL",
+  "amevn": "AM",
+  "ancur": "CW",
+  "aolad": "AO",
+  "asppg": "AS",
+  "atvie": "AT",
```

Comment:
Suggestion: you can eliminate almost all of these entries by filtering based on the first two letters in the BCP-47 ID. (I assume it's still possible for a time zone to change regions due to war so we probably need to keep this table)